google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/vcs/CloneCloudRepositoryDialog.java

```diff
@@ -235,6 +235,7 @@ protected void textChanged(DocumentEvent event) {
         repositorySelector.setCloudProject(projectSelector.getText());
         repositorySelector.setUser(projectSelector.getSelectedUser());
         repositorySelector.setText("");
+        repositorySelector.loadRepositories(null /*onComplete*/);
```

Comment:
Did you mean the Brian Goetz preso? Wasn't he saying there should be no members of type Optional? Can't remember what his point of view on method arguments was but, interestingly enough, the authors of Java 8 make up good points about using Optionals everywhere.

Indeed, you'd still have to pass null in in the body of the arg-less method. But the API wouldn't require you to pass the null in, which is what I like about it. I don't see how the @Nullable way is cleaner, I actually think it's the opposite -- loadRepositories() is so much cleaner than loadRepositories(null /* onComplete */) :)